src/redux/modules/mdb.js

```diff
@@ -572,6 +572,8 @@ const getDenormContentUnit = (state, id) => {
   return cu;
 };
 
+const nestedGetDenormContentUnit = state => id => getDenormContentUnit(state, id);
```

Comment:
Why do you need this?

Reply:
I sue it [here](https://github.com/Bnei-Baruch/kmedia-mdb/blob/d7b66b566d155f2f397982d13e74ddd8df153318/src/components/Sections/Personal/Bookmarks/Bookmarks/List.js#L44) 
For reuse, and can use on _if_ denorm function with single param _id_